### PR TITLE
Add persistence to MariaDB chart

### PR DESCRIPTION
**Pull request by viglesiasce**

### CODE REVIEW

File: stable/mariadb/templates/pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{- if .Values.persistence.enabled }}
+---
```
Comment (prydonius) [inline stable/mariadb/templates/pvc.yaml]: Can we remove this? Helm already inserts these separators.

File: stable/mariadb/values.yaml [hunk 2]
```diff
@@ -25,3 +25,10 @@ imageTag: 10.1.14-r3
 ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-on-first-run
 ##
 # mariadbDatabase:
+
+## Persist data to a persitent volume
+persistence:
+    enabled: true
```
Comment (prydonius) [inline stable/mariadb/values.yaml]: other YAML files use 2 spaces for indentation, can we change this to 2 spaces too?

**Review by prydonius — CHANGES_REQUESTED** [review]

**Comment by prydonius** [comment]
closes #67

**Comment by prydonius** [comment]
We should also update the README to add the new values

File: stable/mariadb/templates/pvc.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
```
Comment (prydonius) [inline stable/mariadb/templates/pvc.yaml:7]: Should this be `volume.beta` now? Or are we keeping the alpha label to support 1.3?

Comment (viglesiasce) [reply stable/mariadb/templates/pvc.yaml:7]: Unfortunately the annotation did not change but the support did.

Check out the following PR to the docs:
https://github.com/kubernetes/kubernetes.github.io/pull/1311/files#diff-e3ed4a5817ba0330edba5ec941757bd0R416

**Comment by viglesiasce** [comment]
All updates are in per review.

File: incubator/mysql/templates/pvc.yaml [hunk 4]
```diff
@@ -1,5 +1,4 @@
 {{- if .Values.persistence.enabled }}
----
```
Comment (prydonius) [inline incubator/mysql/templates/pvc.yaml:2]: looks like you meant to do this in `stable/mariadb/templates/pvc.yaml`

Comment (viglesiasce) [reply incubator/mysql/templates/pvc.yaml:2]: indeed!!!!

**Review by prydonius — APPROVED** [review]

**Comment by viglesiasce** [comment]
Will reopen another branch to clean up some of the madness in the commits here.

**Pull request closed without merging**